Cargo.toml

```diff
@@ -10,6 +10,7 @@ serde_macros = ["webrender_traits/serde_macros"]
 [dependencies]
 app_units = "0.2.5"
 bit-set = "0.4"
+bincode = "0.5.9"
```

Comment:
@nox I don't see why it couldn't although I thought wr-traits was being moved into webrender soon?